### PR TITLE
feat: add FormKit module

### PR DESCRIPTION
**Pull request by andrew-boyd**
Adds FormKit module for Nuxt 3+, providing seamless integration of the FormKit Library. Install instructions can be found here: https://formkit.com/essentials/installation#with-nuxt

### CODE REVIEW

**Comment by Atinux** [comment]
Looks good to me by looking at https://github.com/formkit/formkit/blob/master/packages/nuxt/src/module.ts

Will wait for @pi0 final review :)

**Review by Atinux — APPROVED** [review]

**Comment by pi0** [comment]
Thanks for PR @andrew-boyd. I've made few suggestions in https://github.com/formkit/formkit/issues/67 but looks good to list.

**Pull request merged**
